docs/deploy/aws.md

```diff
@@ -19,8 +19,16 @@ Expected deployment time: 25 min
 
 {% include_relative includes/prerequisites.md %}
 
-## Creating the Database on AWS RDS
-lakeFS requires a PostgreSQL database to synchronize actions on your repositories.
+## Preparing the Database for the Key Value Store
+lakeFS uses a key-value store to synchronize actions on your repositories. Out of the box, this key-value store can rely on DynamoDB or a PostgreSQL DB. As lakeFS open source, you can also write your own implementation and use any other DB
+The following two sections explain how to setup either DynamoDB or PostgreSQL as key-value backing DB
+
+### DynamoDB on AWS
```

Comment:
I think we should put this second as it is less battle proven than postgres.

Reply:
Makes total sense. Also switched the order of the configuration/docker snippets to reflect the same order